### PR TITLE
New intern leads

### PR DESCRIPTION
**Pull request by waryhermit**

### CODE REVIEW

**Comment by kylemathias** [comment]
Congrats @mappuji and @paulbert ! Lead us to victory!

File: pages/team.md [hunk 1]
```diff
@@ -21,6 +20,7 @@
 ## Inactive Interns
 |**Username**|**Join Date**|**Leave Date**|
 |------------|-------------|--------------|
+|[𝖂𝖆𝖗𝖞𝖍𝖊𝖗𝖒𝖎𝖙](profiles/waryhermit.md) (**Intern Lead**)| 20161213 | 20170428 |
```
Comment (kylemathias) [inline pages/team.md]: Why so fancy? =P

Comment (waryhermit) [inline pages/team.md]: To check if anyone reviews 👮

Comment (kylemathias) [inline pages/team.md]: my reaction adding game sucks =/

**Comment by dogi** [comment]
𝖂𝖆𝖗𝖞𝖍𝖊𝖗𝖒𝖎𝖙 awesome more old german characters ;)

**Pull request merged**